dialog/file_xdg_flatpak.go

```diff
@@ -139,7 +139,7 @@ func convertFilterForPortal(fyneFilter storage.FileFilter) (list []*filechooser.
 		return nil, nil
 	}
 
-	out := &filechooser.Filter{Name: lang.L("Filter")}
+	out := &filechooser.Filter{Name: "Filter"}
```

Comment:
From the API docs I see filter Name is "User-visible name for the filter." - so I'm not sure about this - where will it be shown and/or who will translate it instead?

Reply:
I removed the translation as I was unsure if I added it the right way and I didn't know if it was worth it with a new translation string just for the portal dialogs but I can definitely add it back if you prefer. I also tried to not set the field but that results in an error and no dialog being shown. The idea is to let the user have a set of filters that they can select like one for PNG, one for JPG and so on. There is a select widget that lets the user choose from multiple filters.

Reply:
![image](https://github.com/user-attachments/assets/0a6eea8c-842e-4073-91d2-849fa0258ff5)

Reply:
I added back the translations for the filter. Added the Swedish translation for it as well given that it is the same